webapp/src/utils/validators.ts

```diff
@@ -42,6 +42,13 @@ const requiredStringListValidator = stringListValidator(true)
 const optionalStringListValidator = stringListValidator(false)
 
 const numericStringValidator = (value: string): BasicValidation => {
+    //unluckily 0 is null for js/type script
+    //unluckily, 0 is still number
+    //this validator do not eat null as 0
+    if (value == null) {
+        return "invalid"
+    }
+    value = value.toString();
```

Comment:
I missed this line...value is already a string (see funcion's parameter type), so why call toString()?

Reply:
"if you think null check is necessary, then value should by of type string | null"  
Sorry, cant parse that:(

"I missed this line...value is already a string (see funcion's parameter type), so why call toString()?"
because without that, value of  "0" is considered null. Dont look to me why. Before I did that chahnge, the validator was everything except "0" now it works also with "0" or '0' or `0` or 0 or what the JSck